SCAutolib/models/ca.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
```

Comment:
I think it would be better to put everything CA related in ca.py, doesn't make much sense to have ca.py only for the base class. And local_ca and ipa_ca implimentations will be slightly more readable when you can quickly check the base class for reference.